integ/src/pods-template.yaml

```diff
@@ -31,7 +31,7 @@ spec:
     spec:
       containers:
       - name: nginx
-        image: k8s.gcr.io/nginx-slim:0.8
+        image: nginx:1.21.6
```

Comment:
Update nginx image version to `1.21.6` so that able to create arm/amd pods. Old version Nginx doesn't support arm.